### PR TITLE
fix: locking with bzlmod

### PR DESCRIPTION
**Pull request by thesayyn**
Bzlmod locking was broken as well, this fixes it and reverts some changes from https://github.com/GoogleContainerTools/rules_distroless/pull/70

### CODE REVIEW

**Comment by thesayyn** [comment]
ping @alexeagle

**Review by alexeagle — APPROVED** [review]

File: apt/private/resolve.bzl [hunk 1]
```diff
@@ -85,19 +85,85 @@ def internal_resolve(rctx, yq_toolchain_prefix, manifest, include_transitive):
...
+echo ''
+echo 'Writing lockfile to {workspace_relative_path}' 
+cp $lock {workspace_relative_path}
+
+# Detect which file we wish the user to edit
+if [ -e $BUILD_WORKSPACE_DIRECTORY/WORKSPACE ]; then
+    wksp_file="WORKSPACE"
+elif [ -e $BUILD_WORKSPACE_DIRECTORY/WORKSPACE.bazel ]; then
+    wksp_file="WORKSPACE.bazel"
+else
```
Comment (dragos250) [inline apt/private/resolve.bzl:105]: I believe you should have checked MODULE.bazel too.

Comment (jjmaestro) [reply apt/private/resolve.bzl:105]: I've added this in #88, hopefully it'll be merged soon :)

Comment (dragos250) [reply apt/private/resolve.bzl:105]: Thanks for the quick turnaround, I really appreciate it!

**Pull request merged**